tests/load_test.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import time, asyncio
+from openai import AsyncOpenAI, AsyncAzureOpenAI
+import uuid
+import traceback
+import numpy as np
+
+# base_url - litellm proxy endpoint
+# api_key - litellm proxy api-key, is created proxy with auth
+litellm_client = None
+
+
+async def litellm_completion(args, image_url=None):
+    # Your existing code for litellm_completion goes here
+    try:
+        if image_url:
+            messages = [
+                {
+                    "role": "user",
+                    "content": [
+                        {"type": "image_url", "image_url": {"url": image_url}},
+                        {"type": "text", "text": f"This is a test: {uuid.uuid4()}"},
+                    ],
+                },
+            ]
+        else:
+            messages = [{"role": "user", "content": f"This is a test: {uuid.uuid4()}"}]
+
+        response = await litellm_client.chat.completions.create(
+            model=args.model,
+            messages=messages,
+        )
+        print(response)
+        return response
+
+    except Exception as e:
+        # If there's an exception, log the error message
+        with open("error_log.txt", "a") as error_log:
+            error_log.write(f"Error during completion: {str(e)}\n")
+        return str(e)
+
+
+async def main(args):
+    n = 100  # Total number of tasks
+    batch_size = args.req_per_sec  # Requests per second
+    start = time.time()
+
+    async def run_batch(batch):
+        tasks = []
+        for _ in batch:
+            if args.include_image:
+                # Generate a random dimension for the image
+                y_dimension = np.random.randint(100, 1025)
+                image_url = f"https://placehold.co/1024x{y_dimension}/png"
+                task = litellm_completion(args, image_url)
+            else:
+                task = litellm_completion(args)
+            tasks.append(task)
+        return await asyncio.gather(*tasks)
+
+    all_completions = []
+    for i in range(0, n, batch_size):
```

Comment:
per request -> return latency. qps -> we might to send requests async without waiting for the current ones to finish.